scripts/setup-ci.sh

```diff
@@ -38,3 +38,10 @@ curl -L "https://github.com/The-OpenROAD-Project/OpenLane/archive/refs/tags/${OP
 echo "[INFO] Starting PDKs setup..."
 make -C OpenLane pdk-with-sram -j 1 NPROC=1
 echo "[INFO] PDKs setup complete."
+echo "[INFO] Starting Caravel setup..."
+rm -rf caravel
+mkdir -p caravel
+curl -L "https://github.com/efabless/caravel/archive/refs/tags/mpw-5c.tar.gz" \
```

Comment:
This is downloading mpw-5c, but the tag you set for openlane is mpw-6b, shouldn't they match?

Reply:
tag maybe anything. We need `rtl/defines.v` only with `IO_PADS` parameter needed.